README.md

```diff
@@ -30,33 +30,29 @@ Clone the repository.
 
 | Directory  | Chapter |
 | ------------- | ------------- |
-| Chapter 1 - Introduction | src/main/scala/com/professionalscala/ch1 |
-| Chapter 2 - Functional Programming | src/main/scala/com/professionalscala/ch2 |
-| Chapter 3 - Concurrency | src/main/scala/com/professionalscala/ch3 |
-| Chapter 3 - Scala and Java Collections | src/main/scala/com/wrox/professionalscala/ch03 |
+| Chapter 1 - Introduction | src/main/scala/com/wrox/professionalscala/ch01 |
```

Comment:
@lambdista - Thanks to @Mironor's changes the packages are aligned: https://github.com/Mironor/professional-scala-code/tree/master/src/main/scala/com/wrox/professionalscala

@lambdista -There is open discussion below on your unification of the `sbt` run instructions. Please abandon this PR and raise a new one that is restricted to the just package renames. Continue the run simplication in a separate PR. Sounds good?